pkgs/development/python-modules/vapoursynth/default.nix

```
@@ -0,0 +1,27 @@
+{ vapoursynth, cython, buildPythonPackage }:
+buildPythonPackage {
+  pname = "vapoursynth";
+
+  ##
+  # We can keep the vapoursynth frame-server as is,
+  # and not recompile it by just depending on pkgs.vapoursynth.
+  #
+  # There are many attempts, mostly side hustles, that implement their
+  # own language bindings around the VapourSynth C-API.
+  # This means we want to keep this thing separate.
+  #
+  # They all failed however, as the "ecosystem" neccessary for meaningful
+  # use of vapoursynth doesn't exist for other programming languages.
+  #
+  # Still, conceivably, an application can go the route Aegisub went on
+  # Windows using AviSynth, and use vapoursynth without any
+  # additional bindings for loading videos of different formats.
+  # This presents another reason keeping vapoursynth
+  # without its python bindings might be beneficial.
+  inherit (vapoursynth) src version;
+  buildInputs = [
+    vapoursynth
+    cython
+  ];
```

Maybe also other python tests if they exist.
```suggestion
  ];
  pythonImportsCheck = [ "vapoursynth" ];
```

---

Why not just run the unit-tests? They do in fact import vapoursynth :)